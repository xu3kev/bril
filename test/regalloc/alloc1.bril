../reginterp/alloc1.bril