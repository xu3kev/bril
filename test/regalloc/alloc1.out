16
#load: 0
#store: 0
